src/it/scala/org/broadinstitute/dsde/rawls/WorkspaceSimulation.scala

```diff
@@ -269,4 +281,99 @@ class WorkspaceSimulation extends FlatSpec with WorkspaceApiService with EntityA
   it should "create + list many workspaces" in {
     // TODO
   }
+
+  it should "copy a method config from the method repo" in {
```

Comment:
Keeping `WorkspaceSimulation` pure makes sense.  I can move this to a different integration test.